src/AppDevAPI.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import http from 'http'
+import express, {Application, Router, Request, Response, NextFunction} from 'express';

```

Comment:
Minor, but shouldn't there be a space between`{` and `Application`, resulting in:
```
{ Application, Router, Request, Response, NextFunction } from 'express';
```